runbot_push_pot/static/src/js/runbot_push_translation.js

```diff
@@ -0,0 +1,6 @@
+//Copyright 2019 Therp BV <https://therp.nl>
```

Comment:
I think this file and the empty css file can be removed.

Reply:
agreed. I took a note to look into the CI of this repo, this is horribly complex and fails often, as now.